dev_environment/Vagrantfile

```diff
@@ -91,6 +91,49 @@ Vagrant.configure("2") do |config|
         rsync
     SHELL
 
+    # Install Opentelemetry collector
```

Comment:
Can you make this step as conditional only like if cfg['otel_collector'] == "true". we don't want to enable otel_exporter by default,